HelpAssistHer/help-assist-her
# Routing for mini-app

## Description
The first scaffolding PR for the Mini App!! 
- React scaffolding for mini app
- Add React routing
- Add data to redux
- Display data on page

Note: there are some things that are hard coded for now. All of these files will eventually be updated with styling and to match the designs!

## Test Plan
1. Go to `http://localhost:4000/mini-app`
1. Click the button
1. See a list of pregnancy centers!
